main.cpp

```diff
@@ -424,12 +427,26 @@ static const string testnet_seeds[] = {"testnet-seed.alexykot.me",
                                        "testnet-seed.bluematt.me",
                                        "testnet-seed.bitcoin.schildbach.de",
                                        ""};
+static const string signet_seeds[] = {"seednode.signet.bitcoin.sprovoost.nl",
+                                      ""};
 static const string *seeds = mainnet_seeds;
 
 extern "C" void* ThreadSeeder(void*) {
-  if (!fTestNet){
-    db.Add(CService("kjy2eqzk4zwi5zd3.onion", 8333), true);
+  switch (fChain) {
+    case CHAIN_MAIN: {
+        break;
+    }
+    case CHAIN_TESTNET: {
+        break;
+    }
+    case CHAIN_SIGNET: {
+        db.Add(CService("178.128.221.177", 38333), true);
+        db.Add(CService("v7ajjeirttkbnt32wpy3c6w3emwnfr3fkla7hpxcfokr3ysd3kqtzmqd.onion", 38333), true);
```

Comment:
I'm pretty sure this codebase predates v3 onion addresses...

Reply:
I'm not sure if that matters: Tor 3 uses the same proxy as Tor 2 for outbound connections, so it should connect fine. The main benefit of this onion address is that it's likely to outlive the hardcoded IP.

Reply:
It won't connect because it won't parse.

The `CService` implementation in this codebase comes from 2012 Bitcoin Core, long before the concept of Tor V3 even existed...

Reply:
Ah, I assumed it would just treat it as domain name and not parse it. Will remove then...